### PR TITLE
Update dependencies and API

### PR DESCRIPTION
**Pull request by krzyzanowskim**
- Update deps TextFormation -> 0.9.0, TextStory -> 0.9.0, STTextView -> 0.9.6
- Adapt new API with `TextStoringAdapter`

### CODE REVIEW

File: DemoApp/EditorViewController.swift [hunk 1]
```diff
@@ -21,12 +21,16 @@ final class EditorViewController: NSViewController {
...
+            StandardOpenPairFilter(open: "{", close: "}"),
+			StandardOpenPairFilter(open: "[", close: "]"),
+            StandardOpenPairFilter(open: "(", close: ")"),
 		]
 
+        let indenter = TextualIndenter(patterns: TextualIndenter.basicPatterns)
+
 		let providers = WhitespaceProviders(
-			leadingWhitespace: WhitespaceProviders.passthroughProvider,
+            leadingWhitespace: indenter.substitionProvider(indentationUnit: "    ", width: 4),
```
Comment (krzyzanowskim) [inline DemoApp/EditorViewController.swift]: for some reason that doesn't seem to work as expected. The indentation is not applied realiably

Comment (mattmassicotte) [reply DemoApp/EditorViewController.swift]: hmmmm ok I'll have a look

**Comment by mattmassicotte** [comment]
I'm really sorry you had to deal with all this "adapter" junk. Some of this predates me knowing what I was doing, and I experimented with a few ideas here that didn't work out so well. I'll have a closer look tomorrow.

**Comment by mattmassicotte** [comment]
Thanks for wrestling with this. I definitely have to come up with a better solution for TextFormation here.

**Pull request merged**